src/redux/motorcycles/motorcycles.js

```diff
@@ -0,0 +1,67 @@
+import Axios from 'axios';
+
+const BASE_URL = '';
+
+const FETCH_MOTORCYCLES = 'BOOK-APPOINTMENT/MOTORCYCLES/FETCH_MOTORCYCLES';
+const CREATE_MOTORCYCLE = 'BOOK-APPOINTMENT/MOTORCYCLES/CREATE_MOTORCYCLE';
+const DELETE_MOTORCYCLE = 'BOOK-APPOINTMENT/MOTORCYCLES/DELETE_MOTORCYCLE';
+
+const fetchMotorcycles = (payload) => ({
+  type: 'FETCH_MOTORCYCLE',
+  payload,
+});
+
+const createMotorcycle = (payload) => ({
+  type: 'CREATE_MOTORCYCLE',
+  payload,
+});
+
+const deleteMotorcycle = (payload) => ({
+  type: 'DELETE_MOTORCYCLE',
+  payload,
+});
+
+const motorcyclesState = [];
+
+export const fetchMotorcyclesApi = (accessToken) => async (dispatch) => {
+  const returnValue = await Axios.get(`${BASE_URL}/motorcycles`, {
+    headers: {
+      Authorization: `Bearer ${accessToken}`,
+    },
+  });
+  const { motorcycles } = returnValue.data.data;
+  dispatch(fetchMotorcycles(motorcycles));
+};
+
+export const createMotorcycleApi = (accessToken, data) => async (dispatch) => {
+  await Axios.post(`${BASE_URL}/motorcycles/`, {
+    headers: {
+      Authorization: `Bearer ${accessToken}`,
+    },
+  });
+  dispatch(createMotorcycle(data));
+};
+
+export const deleteMotorcycleApi = (accessToken, id) => async (dispatch) => {
+  await Axios.delete(`${BASE_URL}/motorcycles/${id}`, {
+    headers: {
+      Authorization: `Bearer ${accessToken}`,
+    },
+  });
+  dispatch(deleteMotorcycle(id));
+};
```

Comment:
All API endpoints should be created in a separate file, so you import them into the actions creators and use them. I believe that would make things a little clearer than creating all of them all inside a single file. Thank you. 

You could create a file like `api.js` to contain all your API endpoints.